src/main/java/com/hotline/pageobject/pages/MobilesPage.java

```diff
@@ -15,48 +15,59 @@
     private WebElement categoryChargers;
     private WebElement categoryAccessories;
     private WebElement categoryComponentsForPhones;
+    private WebElement applePhonesLink;
 
     MobilesPage(WebDriver driver) {
         super(driver);
-        headerTitle = driver.findElement(By.xpath("//h1/i[@class='icon-menu icon-menu-mobile']"));
-        categories = driver.findElements(By.cssSelector(".category-navigation-item"));
-        categorySmartphones = driver.findElement(
-                By.xpath("//h2[@data-scroll-smooth-to-element-to='telefony-i-garnitury']"));
-        categoryWatchesBands = driver.findElement(
-                By.xpath("//h2[@data-scroll-smooth-to-element-to='smart-chasy-i-fitnes-braslety']"));
-        categoryChargers = driver.findElement(
-                By.xpath("//h2[@data-scroll-smooth-to-element-to='zaryadka-i-sinhronizaciya']"));
-        categoryAccessories = driver.findElement(
-                By.xpath("//h2[@data-scroll-smooth-to-element-to='dopolnitelnoe-oborudovanie-i-aksessuary']"));
-        categoryComponentsForPhones = driver.findElement(
-                By.xpath("//h2[@data-scroll-smooth-to-element-to='komplektuyuschie-dlya-telefonov-i-smartfonov']"));
     }
 
     public WebElement getHeaderTitle() {
+        headerTitle = driver.findElement(By.xpath("//h1/i[@class='icon-menu icon-menu-mobile']"));
```

Comment:
In general, try to avoid xpath if it is possible, it may help in some cases, but generally css selectors are more than enough.